### ISSUE TITLE
Input.ChoiceSet accessibility issues

### ISSUE THREAD

**Issue by timotiusmargo** [microsoft/AdaptiveCards#2333]
# Platform

* .NET WPF
* UWP

# Author or host

Author

# Version of SDK

Latest master

# Issue

ComboBox has a few accessibility issues.
* No accessible name/label when focused (the issue with not using the Header property, or setting another external element as the label)
* This seems to be due to input not having an affordance to specify a title/label. The "title" property seems to no-op.
* Current selection text isn't read when focused.
* Advanced navigation (CapsLock + Left/RightArrow) into the ComboBox read out a redundant element. Only the deepest element indicated the current selection.

```JSON
Adaptive Card JSON Payload:
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.0",
    "body": [
        {
            "type": "TextBlock",
            "text": "This is intended as the ComboBox label"
        },
        {
            "type": "Input.ChoiceSet",
            "id": "myColor",
            "style": "compact",
            "title": "Is this not the label?",
            "value": "1",
            "choices": [
                {
                    "title": "Red",
                    "value": "1"
                },
                {
                    "title": "Green",
                    "value": "2"
                }
            ]
        }
    ]
}
```

**Comment by shalinijoshi19** [comment]
@Paulcam206, FYI resurrecting this from the backlog as part of the Accessibility work.

**Comment by jonmill** [comment]
This bug should be fixed in the latest release; please re-open if you find that is not the case. Thanks!
